lib/util/SortableSet.js

```diff
@@ -91,10 +108,12 @@ class SortableSet extends Set {
 		return newData;
 	}
 
+	/** @private */
```

Comment:
Just curious – have you considered switching to TypeScript entirely, instead of using doc comments?

```ts
private invalidateCache() {
```

Reply:
Webpack does not want to transpile its code for many reasons 

@TheLarkInn Do you have any docs around that architectural decision in wiki or somewhere?

Reply:
I, too, am interested, because things like `SortFn` would be perfect in a types.d.ts, defined as `type SortFn<T> ...` and then `@typedef`d as need as `/** @typedef {import("types").SortFn<T>} SortFn */`. But I don't know if sequestering this kind of definitions into compile-time-only files fits with the project's style.

Reply:
We have talked about this at length in many threads and if it really is needed we can provide a document. The short of it is we do not want anything except JavaScript.

Reply:
> if it really is needed we can provide a document

please. because 

> we do not want

does not look as argument to me